www/lib/helpers.py

```diff
@@ -144,7 +144,7 @@ def get_image_url(cls, rel_img_path):
     @classmethod
     def save_image(cls, raw_img, rel_img_path):
         if settings.IMAGE_STORAGE_MODE == cls._LOCAL_STORAGE:
-            filename = cls.get_image_url(rel_img_path)
+            filename = os.path.join(settings.IMAGE_STORAGE_LOCAL_DIR, rel_img_path)
```

Comment:
This is NOT correct. You need to use the get_image_url function. Why are you reverting this change?